include/types.h

```diff
@@ -540,6 +540,18 @@ enum class RetainedFieldNumber {
 
 };
 
+enum class DefaultValueFiller {
+    // Fill with "missing" values as default (applicable to most)
+    MISSING = 0,
+
+    // Fill with 0 values as default (applicable to AD)
+    ZERO,
+
+    // Do not fill with default values (FORMAT field will be dropped
+    // when >=1 sample(s) have no values according to VCF spec)
+    NONE
```

Comment:
I'm still unsure if we have any use case for the `NONE` setting. Can we just get rid of it? (sorry our realtime verbal conclusion slipped my mind)